docs/source/python/data.rst

```diff
@@ -264,6 +270,32 @@ individual arrays, and no copy is involved:
    arr.type
    arr
 
+Map arrays
+~~~~~~~~~~
+
+MapArrays can be constructed from lists of lists of tuples (key-item pairs), but only if
+the type is explicitly passed into :py:meth:`array`:
```

Comment:
Not that important, but I _think_ you don't need the :py: in all those cases, since the "currentmodule" is pyarrow, so that should be searched first.

Reply:
Thanks! That's makes things simpler :)